index.js

```diff
@@ -1,5 +1,8 @@
 'use strict';
 var chalk = require('chalk');
+var isBrowser = require('is-in-browser').default;
```

Comment:
I don't think we're interested in specific browser code. This should be solved by things like Webpack or others. But I'll let @sindresorhus decide on this one.